### ISSUE TITLE
Fusion of Convolution and BatchNorm

### ISSUE THREAD

**Issue by hbwx24** [pytorch/pytorch#70487]
### 🚀 The feature, motivation and pitch

Is there an operator that fusion conv and BN here？Such as 
https://docs.nvidia.com/deeplearning/cudnn/developer-guide/index.html#op-fusion
or

https://docs.nvidia.com/deeplearning/cudnn/api/index.html#cudnnFusedOps_t
